README.md

```diff
@@ -75,15 +92,50 @@ class ParserTests {
     void fuzzTest(@NotNull String input) {
         assertEquals(input, SomeScheme.decode(SomeScheme.encode(input)));
     }
+	
+    @FuzzTest
+    void fuzzTest(@NotNull @WithUtf8Length(min=10, max=200) String input, @InRange(min=-10, max=10) int n) {
+        assertEquals(input, SomeScheme.decode(SomeScheme.encode(input)));
+        assertTrue(n >= -10 && n <= 10);
+    }
 }
 ```
+A complete Maven example project can be found in [examples/junit](examples/junit).
+
+
+## Running Jazzer
+
+Jazzer can be run in two ways: using the JUnit integration or via standalone Jazzer binary.
+
+### Using JUnit Integration
+To run fuzz tests, use your build system as you would for regular tests.
+Methods annotated with `@FuzzTest` can be executed in two modes: regression mode and fuzzing mode.
+
+Jazzer manages inputs for fuzz tests using an *inputs directory*.
+By default, this directory is derived from the package and class name of the fuzz test.
+For example, if the fuzz test is in the class `src/test/java/com/example/ValidFuzzTestsInputs.java`, method `byteFuzz`, the crash files directory is located in `src/test/resources/com/example/ValidFuzzTestsInputs/byteFuzz`.
+
+If this directory does not exist, Jazzer will save crash inputs in the directory from which the tests are executed.
```

Comment:
I would move this section below the section fuzzing mode, perhaps even create a seperate section called "Inputs" or "Data directories" or something along the lines.